governing-documents/project-annual-review.md

```diff
@@ -10,81 +10,89 @@ nav_order: 5
 # Project Annual Review
 
 ## Overview
+
 In addition to the [quarterly project updates](./project-updates.md), projects will undergo an annual review by the TAC. This annual review will replace one of the quarterly reports. Unlike the quarterly updates, the annual review process will be a "big picture" assessment of the project by reviewing the progress on goals from the prior year and goals for the next year. In addition, the annual review process will include a deep dive into the project health, including contributor and maintainer diversity and adoption metrics, and will include an assessment of the current [project lifecycle state](./project-lifecycle.md). The review will result in a set of recommendations for the project to improve and/or recommendation to move a project to another stage.
 
 ## Timeline
-Annual reviews will replace the Q1 quarterly report.  **If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
 
-**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
+Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
+
+**NOTE:** If a project has genuinely stalled the project's TSC can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
```

Comment:
Q: Would project's TSC decide to EOL their project or do they bring it to LFDT TAC for the move?

Reply:
I believe the TSC has the governance authority to do it. The mechanics would be situational. This sentence is a cleanup / clarification (at least it clarified it for me) on what was in the document already.